airbyte-webapp-e2e-tests/cypress/e2e/connection/connectionStreamDetails.cy.ts

```diff
@@ -0,0 +1,141 @@
+import {
+  getConnectionCreateRequest,
+  getPostgresCreateDestinationBody,
+  getPostgresCreateSourceBody,
+  requestCreateConnection,
+  requestCreateDestination,
+  requestCreateSource,
+  requestDeleteConnection,
+  requestDeleteDestination,
+  requestDeleteSource,
+  requestSourceDiscoverSchema,
+  requestWorkspaceId,
+} from "commands/api";
+import { Connection, Destination, DestinationSyncMode, Source, SourceSyncMode } from "commands/api/types";
+import { appendRandomString } from "commands/common";
+import { runDbQuery } from "commands/db/db";
+import {
+  createAccountsTableQuery,
+  createUserCarsTableQuery,
+  dropAccountsTableQuery,
+  dropUserCarsTableQuery,
+  getCreateUsersTableQuery,
+  getDropUsersTableQuery,
+} from "commands/db/queries";
+import { initialSetupCompleted } from "commands/workspaces";
+import * as connectionPage from "pages/connection/connectionPageObject";
+import { StreamDetailsPageObject } from "pages/connection/streamDetailsPageObject";
+import { NewStreamsTablePageObject } from "pages/connection/streamsTablePageObject/NewStreamsTablePageObject";
+
+const dropTables = () => {
+  runDbQuery(
+    getDropUsersTableQuery("users"),
+    getDropUsersTableQuery("users2"),
+    dropAccountsTableQuery,
+    dropUserCarsTableQuery
+  );
+};
+
+describe.skip("Connection - Stream details", () => {
+  const streamsTable = new NewStreamsTablePageObject();
+  const streamDetails = new StreamDetailsPageObject();
+
+  let source: Source;
+  let destination: Destination;
+  let connection: Connection;
+
+  before(() => {
+    dropTables();
+
+    runDbQuery(
+      getCreateUsersTableQuery("users"),
+      getCreateUsersTableQuery("users2"),
+      createAccountsTableQuery,
+      createUserCarsTableQuery
+    );
+
+    initialSetupCompleted();
+
+    requestWorkspaceId().then(() => {
+      const sourceRequestBody = getPostgresCreateSourceBody(appendRandomString("Sync Mode Test Source"));
+      const destinationRequestBody = getPostgresCreateDestinationBody(appendRandomString("Sync Mode Test Destination"));
+
+      return requestCreateSource(sourceRequestBody).then((sourceResponse) => {
+        source = sourceResponse;
+        requestCreateDestination(destinationRequestBody).then((destinationResponse) => {
+          destination = destinationResponse;
+        });
+
+        return requestSourceDiscoverSchema(source.sourceId).then(({ catalog, catalogId }) => {
+          const connectionRequestBody = getConnectionCreateRequest({
+            name: appendRandomString("Sync Mode Test connection"),
+            sourceId: source.sourceId,
+            destinationId: destination.destinationId,
+            syncCatalog: catalog,
+            sourceCatalogId: catalogId,
+          });
+          return requestCreateConnection(connectionRequestBody).then((connectionResponse) => {
```

Comment:
Would it be worth wrapping this into a promise so we could actually await them?

Reply:
I've tried before cypress doesn't play well with async/await. These are not promises but cypress chains, which use the same interface as promises.